Dockerfile.keyserver

```diff
@@ -9,5 +9,5 @@ LABEL org.label-schema.schema-version="1.0"
 LABEL org.label-schema.build-date=$BUILD_DATE
 LABEL org.label-schema.vcs-url="https://github.com/EGA-archive/LocalEGA"
 LABEL org.label-schema.vcs-ref=$SOURCE_COMMIT
-COPY --from=builder ega-data-api-keyserver/target/ega-data-api-keyserver-0.0.1-SNAPSHOT.jar .
+COPY --from=builder ega-data-api-key/target/ega-data-api-keyserver-0.0.1-SNAPSHOT.jar .
```

Comment:
Actually from https://github.com/EGA-archive/ega-data-api/blob/master/ega-data-api-key/pom.xml#L11  it's ega-data-api-key-0.0.1-SNAPSHOT.jar right?